contracts/utils/RSKAddrValidator.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier:MIT
```

Comment:
I suppose we should only import [it from the contracts](https://github.com/rsksmart/rif-relay-contracts/blob/main/contracts/utils/RSKAddrValidator.sol), shouldn't we?

Reply:
done